SUPPORT.md

```diff
@@ -0,0 +1,38 @@
+## Kubernetes User Support
+
+Welcome to Kubernetes! We use github for tracking issues and feature requests; so
+this isn't the right place to get help, but the following resources are available
+below, thanks for understanding. 
+
+### Stack Exchange
+
+The Kubernetes Community is active on Stack Overflow, you can post your questions there: 
+
+* [Kubernetes on Stack Overflow](http://stackoverflow.com/questions/tagged/kubernetes)
+
+  * Here are some tips for [about how to ask good questions](http://stackoverflow.com/help/how-to-ask).
+  * Don't forget to check to see [what's on topic](http://stackoverflow.com/help/on-topic).
+
+### Real-time Chat
+
+* [Slack](https://kubernetes.slack.com) ([registration](http://slack.k8s.io)):
+The `#kubernetes-users` and `#kubernetes-novice` channels are usual places where 
+people offer support.
+
+* Also check out the 
+[Slack Archive](http://kubernetes.slackarchive.io/) of past conversations.
+
+### Mailing Lists/Groups
+
+* [Kubernetes-users group](https://groups.google.com/forum/#!forum/kubernetes-users)
+
+
+### Documentation 
```

Comment:
Added all your feedback except this one in order to keep the same order as the support rotation document that this is based on. We should perhaps talk about what the order of support escalation should be and then update both documents.